modules/gem5/_posts/2000-03-01-assignment1.md

```diff
@@ -130,7 +121,44 @@ Use `4GHz` as the clock frequency.
 
 In your report, answer the same questions after simulation supported with data. A complete set of simualtion data for this step should include **6 configurations** (2 options for CPU model * 3 options for memory model).
 
-### Step III: General questions
+### Step III: Using different compiler optimizations
+
+In this step, you'll be using different compiler optimizations to run the same
+matrix multiplication program. Compiler optimization flags are options that can
+be used to improve the *performance* of the program at the expense of
+compilation time and the ability to debug the program. The default version of
+the matrix multiply program is compiled with the flag `-O2`:
```

Comment:
Please use one sentence per line instead of hard wrapping.

Reply:
okay.